### PR TITLE
Added Category and Product entity

### PR DESCRIPTION
**Pull request by RamonaMihaela**

### CODE REVIEW

File: src/main/java/com/quickorder/entity/Category.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    @Column(name = "category_name")
+    private String name;
+
+    @Column(name = "category_description")
+    private String description;
+
+    public Long getId() {
+        return id;
+    }
+
```
Comment (kinkey) [inline src/main/java/com/quickorder/entity/Category.java:23]: Please remove unnecessary empty lines.

**Review by BogdanCinar — COMMENTED** [review]
la products trebuie sa avem si o relatie cu subcategorie

in categorie sa avem o relatie de genu:
    @OneToMany(mappedBy="category")
    private List<SubCategory> subCategories;

fara getter si setter daca folosim lombok aici intervine Geo sa ne spuna cum

trebuie hotarat daca face tabela intermediara intre produs si ingredient si modificata clasa de produs in cazul asta

**Review by kinkey — COMMENTED** [review]
We should start extracting repeating strings to constants. for example:
We could have a EntityConstants class that contains the value of the "seq_gen" string.  In addition, we could do the same for other repeating strings, such as description.
Additionally, maybe we should extract the other strings to constants in the class.
For example, in the ingredient class we could have a string called 
`private static final String INGREDIENT = "ingredient_"` and another called `private static final String NAME`which reside in the BaseEntity class.
And in your Ingredient class would could have `private static String INGREDIENT_NAME = INGREDIENT+NAME`, thus reusing existing references.
Even better, we could slightly change the table columns, so that we don't call them ingredient_name, but just name. that way we could highly reuse code.
Thoughts?

**Review by kinkey — APPROVED** [review]
Looks good to me now

**Pull request merged**